src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrTopicOverviewTaskTest.java

```diff
@@ -0,0 +1,148 @@
+        false,
+        new String[] {
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet-pizza-108.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---schokolade-102.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet-kaese--102.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---camping-102.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---kaffee-102.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---kreuzfahrt-102.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet-pizza-100.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet-kueche-was-kosten-herd--co-100.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet-kaese--100.html"
+        }
+      },
+      {
+        0,
+        "overview url containing additional overview urls but no subpages configured",
+        "http://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/index.html",
+        "/wdr/wdr_topic_overview_with_child.html",
+        "",
+        "",
+        false,
+        new String[] { 
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/video-lokalzeit-ruhr--134.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit/video-lokalzeit-am-samstag-240.html",
+        }
+      },
+      {
+        1,
+        "overview url containing additional overview urls with subpages configured",
+        "http://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/index.html",
+        "/wdr/wdr_topic_overview_with_child.html",
+        "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/lokalzeit-ruhr-beitraege-100.html",
+        "/wdr/wdr_topic_overview_child_page.html",
+        false,
+        new String[] { 
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/video-lokalzeit-ruhr--134.html", 
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit/video-lokalzeit-am-samstag-240.html",
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/video-seepferdchen-pruefung-in-muelheim-100.html",
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/video-ruhr-uni-untersucht-umarmungen-100.html",
+          "https://www1.wdr.de/mediathek/video/sendungen/lokalzeit-ruhr/video-aussergewoehnliche-affen-portraits--100.html"
+        }
+      }
+    });
+  }
+  
+  private final String topic;
+  private final String requestUrl;
+  private final String htmlFile;
+  private final String childUrl;
+  private final String childHtmlFile;
+  private final boolean isFileUrl;
+  private final TopicUrlDTO[] expectedUrls;
+  
+  public WdrTopicOverviewTaskTest(final int aMaxSubpages,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)